docs/public-api.asciidoc

```diff
@@ -302,6 +302,25 @@ NOTE: Return value of <<convenient-capture-span>> method overloads that accept T
 
 NOTE: Code samples above use `Elastic.Apm.Agent.Tracer.CurrentTransaction`. In production code you should make sure the `CurrentTransaction` is not `null`.
 
+[float]
+[[api-transaction-Custom]]
+==== `Dictionary<string,string> Custom`
+
+Custom context is used to add non-indexed, custom contextual information to transactions.
+Non-indexed means the data is not searchable or aggregatable in Elasticsearch, and you cannot build dashboards on top of the data.
+However, non-indexed information is useful for other reasons, like providing contextual information to help you quickly debug performance issues or errors.
+
+Unlike <<api-transaction-tags>>, the data in this property is not trimmed.
+
+[source,csharp]
+----
+Agent.Tracer.CaptureTransaction(transactionName, transactionType, (transaction) =>
+{
+	transaction.Custom.Add("foo", "bar");
```

Comment:
It might be more consistent to use indexer here just as in `Labels` example and also it might be worth using value that would not be possible to store as label (i.e., very long string).

Reply:
Makes sense, changed it to indexer

Reply:
Makes sense, changed it to indexer